modules/tinymce/src/themes/silver/main/ts/api/Options.ts

```diff
@@ -215,6 +215,11 @@ const register = (editor: Editor): void => {
     processor: 'object'
   });
 
+  registerOption('ui_mode', {
+    processor: 'string',
+    default: 'default'
```

Comment:
Is the name `default` really good here in case we change the default. Better to call the options `split` and `same` or something. Not sure hard to name this properly.

Reply:
how about `combined`

Reply:
Renaming to `combined` 👍